lesson05/assignment/css/style.scss

```diff
@@ -1,6 +1,210 @@
 // Import partials here
-@import 'base.normalize';
-@import 'settings.variables';
-@import 'settings.responsive';
-@import 'components.content';
+// @import 'base.normalize';
+// @import 'settings.variables';
+// @import 'settings.responsive';
+// @import 'components.content';
 
+* {
+    margin: 0;
+    padding: 0;
+}
+
+.home {
+    display: grid;
+    grid-template-areas: "one one"
+        "two two"
+        "three four"
+        "five five"
+    ;
+
+    min-height: 100vh;
+    grid-template-columns: 1fr 3fr;
+}
+
+.one {
+    grid-area: one;
+}
+
+.two {
+    grid-area: two;
+}
+
+.three {
+    grid-area: three;
+}
+
+.four {
+    grid-area: four;
+}
+
+.five {
+    grid-area: five;
+
+}
```

Comment:
Clearer names for these classes and grid areas would make this code easier to read and maintain. What about heaser, nav, aside, main, and footer?